docs/resources/virtual_machine.md

```diff
@@ -130,40 +130,40 @@ disable_root: false
 
 Required:
 
-- `disk` (Block List, Min: 1) (see [below for nested schema](#nestedblock--devices--disk))
-- `interface` (Block List, Min: 1) (see [below for nested schema](#nestedblock--devices--interface))
+- `disk` (List of List of Object)
+- `interface` (List of List of Object)
```

Comment:
I'm tagging @cloudmaniac for help here. I lack experience with VMware so I can't tell if these descriptions are helpful. As a non-VMware I can't say I know what "List of Object" is or would be. @nikchern

Reply:
I'm not sure. `List of List of Object` doesn't make sense to me.
Maybe more something like: `(Required) Specification for virtual disks on the virtual machines. See XYZ for more information.`?